packages/app/obojobo-express/routes/api/drafts.js

```diff
@@ -189,7 +189,7 @@ router
 // mounted as /api/drafts
 router
 	.route('/')
-	.get(requireCanViewDrafts)
+	.get(requireCanPreviewDrafts)
```

Comment:
I like the new `canPreviewDrafts`, this is the only case where it feels a bit weird to use. In other words, this is "If user X can preview drafts then return a list of drafts that user X has".

I'm thinking do we even need this check here? Could we always let users get a list of their drafts? Students (as the system is) would simply get nothing returned as they don't have drafts.